components/elm/src/main/ncdio_pio.F90.in

```diff
@@ -255,7 +258,8 @@ contains
        end if
     end if
 
-    ierr = pio_createfile(pio_subsystem, file, my_io_type, fname, ior(PIO_CLOBBER,PIO_64BIT_OFFSET))
+    mode = shr_pio_getioformat(lnd_id)
```

Comment:
`mode` could be renamed to `io_mode`

Reply:
Out of curiosity, what is the netcdf format (instead of `64BIT_OFFSET`) that works the large restart file?

Reply:
Thanks for the suggestions, @bishtgautam. I opt to add a comment by the declaration but keep the variable name 'mode'

For ne1024pg2, have to use 64bit_data. The use of 8-byte integers is also necessary for this resolution to store the values of several variables (or expressions) as shown below. The original line of code above - pio_createfile also loses the ability to respond to the value set via PIO_NETCDF_FORMAT.